net.tangysoft.usenext.yaml

```diff
@@ -0,0 +1,64 @@
+app-id: net.tangysoft.usenext
+runtime: org.gnome.Platform
+sdk: org.gnome.Sdk
+runtime-version: '46'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.mono6
+rename-desktop-file: usenext.desktop
+command: usenext
+finish-args:
+  - --share=network
+  - --socket=x11
+  - --share=ipc
+  - --filesystem=home
+  - --env=MONO_MANAGED_WATCHER=dummy
+build-options:
+  append-path: /usr/lib/sdk/mono6/bin
+  append-ld-library-path: /usr/lib/sdk/mono6/lib
+  env:
+    PKG_CONFIG_PATH: /app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/mono6/lib/pkgconfig
+modules:
+  - shared-modules/gtk2/gtk2.json
+
+  - name: mono-runtime
+    buildsystem: simple
+    build-options:
+      no-debuginfo: true
+      strip: true
+    build-commands:
+      - /usr/lib/sdk/mono6/install.sh
+
+  - name: gtk-sharp
+    sources:
+      - type: git
+        url: https://github.com/mono/gtk-sharp.git
+        commit: 3440993e00c89791c5401ae0dcc7508d81347739
+      - type: patch
+        path: gtk-sharp2-2.12.12-glib-include.patch
+      - type: shell
+        commands:
+          - ./bootstrap-2.12 --prefix=/usr
```

Comment:
I prefix is not `/usr`

Reply:
hm. maybe. I've taken that from Bless: https://github.com/flathub/com.github.afrantzis.Bless/blob/28dfa6b7430d5b143123943bec453a997fd94981/com.github.afrantzis.Bless.yaml#L40

Do you still think this should be /app?

Reply:
I don't know, but if `prefix` mean what it mean everywhere else, I think so. If it mean something else, then who knows...

Reply:
Let's ask @FakeShemp who authored this: https://github.com/flathub/com.github.afrantzis.Bless/commit/5512da8ea6105027844b8e4baeb4856edea6a607

Let me resolve for now as it works well enough for me.

Reply:
can you try making this `/app`? according to the log it is being used as "installation prefix" which means it should be /app.

The library still gets installed to /app probably because it is using something else.